java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/symbol/ConstructorCallSymbolProvider.java

```diff
@@ -32,11 +42,50 @@ public List<SymbolInformation> get(SearchMatch match) throws CoreException {
             }
             symbol.setContainerName(mod.getParent().getElementName());
             symbol.setLocation(getLocation(mod, match));
-            symbols.add(symbol);
+            if (this.query.contains(".")) {
+                ICompilationUnit unit = mod.getCompilationUnit();
+                ASTParser astParser = ASTParser.newParser(AST.getJLSLatest());
+                astParser.setSource(unit);
+                astParser.setResolveBindings(true);
+                CompilationUnit cu = (CompilationUnit) astParser.createAST(null);
+                cu.accept(new ASTVisitor() {
+                    // we are only doing this for MethodInvocation right now
+                    // look into MethodDeclaration if needed
+                    public boolean visit(MethodInvocation node) {
+                        try {
+                            IMethodBinding binding = node.resolveMethodBinding();
+                            if (binding != null) {
+                                // get fqn of the method being called
+                                ITypeBinding declaringClass = binding.getDeclaringClass();
+                                if (declaringClass != null) {
+                                    String fullyQualifiedName = declaringClass.getQualifiedName() + "." + binding.getName();
+                                    // match fqn with query pattern
+                                    if (fullyQualifiedName.matches(getCleanedQuery(query))) {
```

Comment:
what does getCleanedQuery do?

Reply:
@shawn-hurley updated...but search queries like `get*` are not valid regexes when it comes to Java regexes. So when matching the fqn, we need to make sure `get*` converts to `get.*` so it java regex works as expected.